libs/dal/src/repositories/user/user.schema.ts

```diff
@@ -11,6 +11,7 @@ const userSchema = new Schema(
     profilePicture: Schema.Types.String,
     resetToken: Schema.Types.String,
     resetTokenDate: Schema.Types.Date,
+    onBoarding: Schema.Types.Boolean,
```

Comment:
```suggestion
    completedOnBoarding: Schema.Types.Boolean,
```

A small suggestion regarding the schema, maybe worth considering using "completedOnboarding" or shoOnboarding to better give context to what this fields does?